draft/2022-08-03-this-week-in-rust.md

```diff
@@ -28,20 +28,41 @@ and just ask the editors to select the category.
 -->
 
 ### Official
+* [Increasing the glibc and Linux kernel requirements](https://blog.rust-lang.org/2022/08/01/Increasing-glibc-kernel-requirements.html)
 
 ### Foundation
 
 ### Newsletters
 
 ### Project/Tooling Updates
+* [rust-analyzer changelog #140](https://rust-analyzer.github.io/thisweek/2022/08/01/changelog-140.html)
+* [GCC Rust Monthly Report #19 July 2022](https://thephilbert.io/2022/08/01/gcc-rust-monthly-report-19-july-2022/)
+* [This Month in hyper: July 2022](https://seanmonstar.com/post/691039972381294592/this-month-in-hyper-july-2022)
+* [Bevy 0.8](https://bevyengine.org/news/bevy-0-8/)
 
 ### Observations/Thoughts
+* [Rust linux kernel development](https://www.jackos.io/rust-kernel/rust-for-linux.html)
+* [Proc macro support in rust-analyzer for nightly rustc versions](https://fasterthanli.me/articles/proc-macro-support-in-rust-analyzer-for-nightly-rustc-versions)
+* [Manage keybindings in a Rust terminal application](https://dystroy.org/blog/keybindings/)
+* [Safety](https://nested.substack.com/p/safety)
+* [Paper Review: Safe, Flexible Aliasing with Deferred Borrows](https://blog.theincredibleholk.org/blog/2022/07/29/safe-flexible-aliasing-with-deferred-borrows/)
+* [Uncovering a Blocking Syscall](https://chrissardegna.com/blog/uncovering-a-blocking-syscall/)
+* [nt-list: Windows Linked Lists in idiomatic Rust](https://colinfinck.de/posts/nt-list-windows-linked-lists-in-idiomatic-rust/)
 
 ### Rust Walkthroughs
+* [Living with the Rust trademark](https://lwn.net/SubscriberLink/901816/000516eb2cb47e05/)
```

Comment:
FWIW, I saw this last week, but decided not to include it because at the root of the story is just one person being a pain and trying to stir up a lot of conflict.

Thankfully not much conflict followed, but is it in the Rust community's interest to bring more attention to this story?

@nellshamrell any thoughts?

Reply:
Had to think about this one a bit. If it were just an exploration of the trademark, I would favor including it. However, the comments/discussion on it (and around it) are pretty toxic and I don't think there would be a benefit to drawing more attention to it.